### DISCUSSION TITLE
Recent builds not published?

### DISCUSSION THREAD

**Discussion by xxleyi** [johnlindquist/kit#766] [General]
I found kit had some new builds, but in website download link is still `1.11.1`
![image](https://user-images.githubusercontent.com/12626454/167115358-3703b8a3-e0bb-4ec0-a527-f4e39d51ffc5.png)

**Comment by johnlindquist** [comment]
If you'd like to help test the private builds, you can join us on discord:

https://discord.gg/44smSDRT
